Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HARRY.MSG

```diff
@@ -109,27 +109,23 @@
 {1000}{}{Master|Mutant Leader}
 {1001}{}{Lou|Lieutenant|Lutenant|Loo|Lou Tenant|Lt}
 {1002}{}{Set|Seth|Wuss}
-{1003}{}{Water Shed|Watershed|Shed}
+{1003}{}{Water Shed|Watershed|Shed|Necropolis|CITY OF DEATH|CITY OF THE DEAD}

```

Comment:
City of dead was on purpose, but I guess it doesn't matter. Nobody will type it in anyways.